Bell'sInequality43.md

```diff
@@ -0,0 +1,12 @@
+History:
+In 1935, Albert Einstein teamed up with Boris Podolsky and Nathan Rosen to publish a paper commonly known by the initials EPR. 
+They argued that Quantum Mechanics (QM) was incomplete because it relied on so-called "Hidden Variables" that could account for some of the uncertainties in QM. Hidden Variables refer to microscopic characteristics of fundamental particles that we cannot currently observe due to technological limitations, although future advancements might change this. 
+In other words, we may need more advanced tools to uncover the details at the smallest scales. Since we cannot currently see these variables, they remain "hidden."
+However, understanding them could clarify the otherwise puzzling behaviour of particles. 
+The Heisenberg Uncertainty Principle (HUP), a fundamental aspect of Quantum Mechanics, posits that these variables not only elude observation but do not exist outside the context of an observation, challenging our conventional understanding of reality.
+EPR provided a clear yet impactful definition of what they termed an "element of reality": if a property of a system can be predicted with complete certainty (100%) without influencing that system, then it must relate to an element of reality. This definition is straightforward and widely accepted. We will use it in our proof of Bell's Theorem.
+EPR argued that either Hidden Variables exist, or the attributes of particles (such as position, velocity, energy, and polarization) are not real or defined until they are measured. 
+This idea aligns with the Heisenberg Uncertainty Principle (HUP). They also claimed that it is "unreasonable" to think that these attributes require observation to be real, thus inferring that Hidden Variables must exist. 
+Einstein famously stated, "I think that a particle must have a separate reality independent of the measurements. 
+That is: an electron has spin, location, and so on, even when it is not being measured. I like to think that the moon is there even if I am not looking at it." While some accepted this view, others, including Bell, disagreed.
```

Comment:
I know it might not be that important to some, but I might suggest modifying this to state that Bell, at first, agreed to this and was actually trying to prove that local hidden variables exist, yet he ended up proving the opposite (technically, he didn't prove the opposite, he just came up with a way to test whether the physical phenomena were locally real or not, and through violating his inequality, that only says that "at least" one of those is not the case). That's to the limit of my knowledge, and, please, correct me if I'm wrong. I'd be delighted to get to know how the accurate scenario went.

Reply:
Let’s discuss that in a new PR once it’s merged.